test.py

```diff
@@ -582,5 +582,55 @@ def test_set_font_family(self):
         self.assertEqual(result, expected_svg)
 
 
+class PrepToolWriterTest(unittest.TestCase):
+    '''PrepToolWriter tests'''
+
+    def setUp(self):
+        xml2rfc.log.quiet = True
+        path = 'tests/input/elements.xml'
+        self.parser = xml2rfc.XmlRfcParser(path,
+                                           quiet=True,
+                                           options=default_options,
+                                           **options_for_xmlrfcparser)
+        self.xmlrfc = self.parser.parse()
+        self.writer = xml2rfc.PrepToolWriter(self.xmlrfc, quiet=True)
+
+    def test_rfc_check_attribute_values_editorial(self):
+        rfc = lxml.etree.fromstring('''
+<rfc
+    submissionType="editorial"
+    category="info">
+</rfc>''')
+        self.writer.root = rfc
+
+        self.writer.check_attribute_values(rfc, None)
+        self.assertEqual(len(self.writer.errors), 0)
+
+    def test_boilerplate_insert_status_of_memo_editorial(self):
+        rfc = lxml.etree.fromstring('''
+<rfc
+    number="9280"
+    ipr="trust200902"
+    submissionType="editorial"
+    category="info">
+</rfc>''')
+        self.writer.options.rfc = True
+        self.writer.root = rfc
+
+        self.writer.boilerplate_insert_status_of_memo(rfc, rfc)
+        self.assertEqual(len(self.writer.errors), 0)
+        self.assertEqual(len(rfc.xpath('./section/name[text()="Status of This Memo"]')), 1)
+        paras = rfc.xpath('./section/t')
+        self.assertIn('This document is not an Internet Standards Track specification; it is', paras[0].text)
```

Comment:
* good that we have this test - I'm not finding where we're testing that for the other boilerplate types?
* I worry that this is a little fragile in that it could pass if chunks of text were somehow reordered? Could this be done with a single assert per paragraph instead?

Reply:
Is this better? https://github.com/ietf-tools/xml2rfc/pull/958/commits/e757754c4816270a212d87e8bb252b30a3121226